src/app/zippy/zippy.component.spec.ts

```diff
@@ -77,6 +77,20 @@ describe("ZippyComponent", () => {
     host.click(".zippy__title");
     expect(".zippy__content").not.toExist();
   });
+
+  it('should toggle the content when pressing "Enter"', () => {
+    host = createHost(`<zippy title="Zippy title"></zippy>`);
+
+    const pressEnter = () => {
+      host.dispatchKeyboardEvent('.zippy__title', 'keyup', 'Enter');
+    };
+
```

Comment:
I think it will be helpful to add the following method:

`host.pressEnter('.zippy__title') // if empty it should be on the tested element `

What do u think? Can you add it, please? ( you can add it here https://github.com/NetanelBasal/spectator/blob/master/src/lib/src/internals.ts#L120)

Reply:
Also, add your details to the contributors [file](https://github.com/NetanelBasal/spectator/blob/master/.all-contributorsrc).

Reply:
is Enter that common to warrant such a method? I don't want to over-pollute the API. I mean, if this is added, next thing to be added is Space etc.